include/SDL3/SDL_stdinc.h

```diff
@@ -1283,7 +1283,7 @@ extern SDL_DECLSPEC void SDLCALL SDL_srand(Uint64 seed);
  * different characteristics and you should pick one of those to meet any
  * serious needs.
  *
- * \returns a random value in the range of [0-SDL_MAX_UINT32].
+ * \returns a random value in the range of [SDL_MIN_SINT32-SDL_MAX_SINT32].
```

Comment:
```suggestion
 * \returns a random value in the range of [0-SDL_MAX_SINT32].
```

Reply:
Actually, is this right @JKaniarz?

Reply:
I think the `SDL_MIN_SINT32` lower bound is correct: we're returning the 32 most significant bits from the state, of which the most significant (sign) bit might be set.

Reply:
Right, but I don't think that's what we want here. rand() should never return a negative number.

Reply:
Then perhaps we need to shift the state 33 bits down instead of 32 bits?

Reply:
Maybe? That's why I pinged  @JKaniarz, he'll know better what the right solution here is.

Reply:
I agree that rand shouldn't return negative. Shifting by 33 bits is the right way to get Sint32 output. But I do have a preference for Uint32 output. See my comment below.